blocks/log.py

```diff
@@ -64,10 +69,3 @@ def previous_row(self):
     @property
     def last_epoch_row(self):
         return self[self.status['_epoch_ends'][-1]]
-
-    def to_dataframe(self):
-        """Convert a log into a :class:`.DataFrame`."""
-        if not PANDAS_AVAILABLE:
-            raise ImportError("The pandas library is not found. You can"
-                              " install it with pip.")
-        return DataFrame.from_dict(self, orient='index')
```

Comment:
> while you are the only person I know of who uses this one-liner

I am curious then what do the others do when they want to analyze the log. Always type `DataFrame.from_dict(...)`?

I don't find the link between long compilation times at the cluster and removal of `to_dataframe` is convincing. `pandas` can be made an optional dependency installed only on explicit request and `to_dataframe` can  gracefully fail when it is not installed.